hw/top_earlgrey/doc/design/index.md

```diff
@@ -146,19 +146,15 @@ See that specification for generic details on this scheme.
 
 #### Chip IO Peripherals
 
-##### Pin Multiplexor and Pad Control Module (`pinmux`)
+##### Pin Multiplexor Module (`pinmux`)
 
-**TODO: this section needs to be updated to reflect the pinmux/padctrl merger**
-
-The pin multiplexor's purpose is to route between peripherals and the available multiplexable IO (`MIO_00 .. MIO_31`) of the chip.
+The pin multiplexor's purpose is to route between peripherals and the available multiplexable IO of the chip.
```

Comment:
s/multiplexor/multiplexer? 
https://en.wikipedia.org/wiki/Multiplexer

Reply:
Ha! I just realized the wiki does mention multiplexor as an alternate spelling....